pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -799,13 +802,28 @@ func (b *Build) rebasePod(buildContext BuildContext, images BuildPodImages) (*co
 	}, nil
 }
 
+func (b *Build) completionContainer(images BuildPodImages, buildContext BuildContext, args []string, env []corev1.EnvVar, volumeMounts []corev1.VolumeMount) corev1.Container {
```

Comment:
Can we inline this method now that it doesn't have any special logic?

Reply:
I changed it a bit but I don't think it's worth putting back as it is in 2 places and has a little extraction value

Reply:
Fair. I think there is a bit of stylistic discussion to have here. All the other container specs are defined inline here to make it "easy" to see the defined pod all at once as opposed to navigating into methods. We have gone back and force on the completion container a few times here.